src/service/interfaces.ts

```diff
@@ -9,6 +9,7 @@ export interface BaseServiceInterface {
     lastChange: number;
     controlEnable: CommandEnableInterface;
     currentProcedure: string | undefined;
+    peaId: string;
```

Comment:
peaID is related to a spezific instance, why should it be part of base service interface e.g. also including virtual polServices without peaID

Reply:
Thank you for the hint. To start a (normal) service, a peaId is needed, so it's nice to have. I suggest to add it to ServiceInterface instead of BaseServiceInterface.

At specially in service-launcher the assignment (which service belongs to which pea) kinda gets lost. Simplest solution was to add peaID to the Interface.